inference-engine/src/mkldnn_plugin/utils/debug_capabilities.h

```diff
@@ -4,7 +4,44 @@
 #pragma once
 
 #ifdef CPU_DEBUG_CAPS
-#   define ENABLE_CPU_DEBUG_CAP(_x) _x;
-#else
-#   define ENABLE_CPU_DEBUG_CAP(_x)
-#endif
+
+#include <map>
+#include <string>
+#include <vector>
+
+#define ENABLE_CPU_DEBUG_CAP(_x) _x;
+
+namespace MKLDNNPlugin {
+namespace DebugCaps {
+
+class Config {
+public:
+    Config() {
+        readParam(blobDumpDir, "OV_CPU_BLOB_DUMP_DIR");
+        readParam(blobDumpFormat, "OV_CPU_BLOB_DUMP_FORMAT");
+        readParam(blobDumpNodeExecId, "OV_CPU_BLOB_DUMP_NODE_EXEC_ID");
+        readParam(blobDumpNodeType, "OV_CPU_BLOB_DUMP_NODE_TYPE");
+        readParam(blobDumpNodeName, "OV_CPU_BLOB_DUMP_NODE_NAME");
+        readParam(graphDumpPath, "OV_CPU_GRAPH_DUMP_PATH");
```

Comment:
Still not sure we need to unite PATH and TYPE into single variable. We might want to extend the TYPE for `dot` case and still need path.

Reply:
Agree, but actually the reason I rewrote it into single variable is to prevent adding a new entity for mkldnn_graph, like DUMP_TYPE (new enum basically only for debugging).
Maybe we could stick with the single variable and differentiate types by:
- cout
- *.xml
- *.dot
- everything else -> incorrect value
?

Reply:
Kind of tradeoff: From usage point of view, single variable is better. From source code point of view, better to have separate two.